composer.json

```diff
@@ -39,17 +39,17 @@
         "symfony/polyfill-uuid": "^1.13.1"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.17",
+        "friendsofphp/php-cs-fixer": "^3.4",
```

Comment:
Adding this will disallow any Symfony < 4.4, did we check for this in the CI prefer-lowest jobs?

Reply:
Right, I didn't think about this. I relaxed the constraint to allow both version `2.x` and `3.x` to be installed